5 - RPG/Code/RPG.CharacterBuilder/PlayerCharacter.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace RPG.CharacterBuilder
+{
+    class PlayerCharacter
+    {
+        public CharacterAbilities Abilities { get; internal set; }
+        public CharacterAttributes Attributes { get; internal set; }
+        public PlayerCharacter()
+        {
+            Abilities = new CharacterAbilities();
+            Attributes = new CharacterAttributes();
+        }
+        public PlayerCharacter(CharacterAbilities abilties, CharacterAttributes attributes)
+        {
+            Abilities = abilties;
+            Attributes = attributes;
+        }
+        public PlayerCharacter(CharacterAbilities abilities)
+        {
+            Abilities = abilities;
+            Attributes = new CharacterAttributes();
+        }
+        public PlayerCharacter(CharacterAttributes attributes)
+        {
+            Abilities = new CharacterAbilities();
+            Attributes = attributes;
+        }
```

Comment:
The requirements state that you need attributes before moving onto abilities. Allowing a CTOR like this would allow for some easy bugs to occur.

To be clear, overall I actually really like this class but these two CTOR's should not exist.